packages/runtime/plugin-garfish/package.json

```diff
@@ -81,7 +81,9 @@
     "@swc/helpers": "0.5.1"
   },
   "peerDependencies": {
-    "@modern-js/runtime": "workspace:^2.32.1"
+    "@modern-js/runtime": "workspace:^2.32.1",
+    "react": ">=16",
+    "react-dom": ">=16"
```

Comment:
Maybe we should limit the minimal version to React 17 to keep consistent with other runtime plugins. @nyqykk